app/gradle.properties

```diff
@@ -2,4 +2,6 @@
 VERSION_MAJOR=1
 VERSION_MINOR=14
 VERSION_HOTFIX=0
+VERSION_PATCH=0
+# don't change this value, this will be auto-generated
 VERSION_BUILD=114002
```

Comment:
why do we need this value then?

Reply:
This field will be used for testing purposes. 
When we'll send the feature/release TEST build for QA, they will see the `APP_NAME-VERSION_BUILD` in the build's name.

This will help us to verify what version has bugs, for instance.
The VERSION_BUILD should be generated by github CI. Our CI should commit and push the changes in that file. 
I'm worknig on it